2-mandatory/3-financial-times.js

```diff
@@ -23,15 +33,29 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+     let arr=[]
+    for (let article of allArticleTitles){
+        for (let char of article){
+            if (char>="0" && char<="9"){
```

Comment:
this is nice! Another way to check it, would be to see if any character can be converted into an integer.
!!parseInt("a")

the double ! (bang) would return a boolean value